<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">window-&gt;setStyleSheet(&quot;background-image: url(:/menu/menupic.jpg);&quot;);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButtonmana">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>96</y>
          <width>160</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>管理管理员</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtoncheckfeedback">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>321</y>
          <width>160</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>查看用户反馈</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtondishes">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>209</y>
          <width>160</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>管理菜系</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonunsign">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>433</y>
          <width>160</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>退出登入</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;宋体&quot;;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>最后登入时间：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTime">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>531</width>
          <height>351</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>方正舒体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border: none;
color: white;
</string>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotDotLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>用户1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户4</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>用户名</string>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>密码</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignJustify|AlignBottom</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButtonsnew">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>440</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>添加管理员</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonbacksave">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>440</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>返回上一级并保存</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="DishWidget" name="widgetmenu" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>711</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonbacksave_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>460</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>返回上一级并保存</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonsnewdish">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>460</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>添加菜品</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>450</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>清空文本并返回</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>470</y>
          <width>140</width>
          <height>60</height>
         </rect>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>551</width>
          <height>371</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border: none;
color: white;
</string>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotDotLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>用户1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>用户4</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>用户</string>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>意见</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignJustify|AlignBottom</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>40</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;华文隶书&quot;;</string>
        </property>
        <property name="text">
         <string>当前意见条数：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DishWidget</class>
   <extends>QWidget</extends>
   <header location="global">dishwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
